src/core/api/replication.go

```diff
@@ -78,16 +81,27 @@ func (r *ReplicationAPI) Post() {
 		return
 	}
 
-	if err = startReplication(replication.PolicyID); err != nil {
+	opUUID, err := startReplication(replication.PolicyID)
+	if err != nil {
 		r.HandleInternalServerError(fmt.Sprintf("failed to publish replication topic for policy %d: %v", replication.PolicyID, err))
 		return
 	}
 	log.Infof("replication signal for policy %d sent", replication.PolicyID)
+
+	r.Data["json"] = api_models.ReplicationResponse{
+		UUID: opUUID,
+	}
+	r.ServeJSON()
 }
 
-func startReplication(policyID int64) error {
-	return notifier.Publish(topic.StartReplicationTopic,
+// startReplication triggers a replication and return the uuid of this replication.
```

Comment:
@reasonerjt Is there a reason that we don't log here that we've started a replication job (and in this case explicitly state the UUID of the job)?

Reply:
One replication job is designed to handle replication of one repository,
and one replication action, given certain filter, will map to multiple jobs in job service.
And this PR make it possible to trace a job based on replication action.